investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/repository/model/reporting/InvestigationReporting.java

```diff
@@ -87,6 +87,13 @@ public class InvestigationReporting {
     private String notificationAddTime;
     private String notificationRecordStatusCd;
     private String notificationLastChgTime;
+    private Long caseManagementUid;
+    private Long nacPageCaseUid;
```

Comment:
Hope we do have a corresponding update to the liquibase scripts

Reply:
I believe they were updated in the 6.0.15.1 PR for Liquibase but the model was not updated. This makes sure the columns are getting updated.